### PR TITLE
Restore default pragmas in preset-react for classic runtime

### PR DESCRIPTION
**Pull request by existentialism**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          | Y
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

Ref: https://github.com/babel/babel/issues/11321, https://github.com/mdx-js/mdx/issues/990, https://github.com/gatsbyjs/gatsby/issues/22500

### CODE REVIEW

File: packages/babel-preset-react/src/index.js [hunk 1]
```diff
@@ -8,16 +8,22 @@ import transformReactJSXSelf from "@babel/plugin-transform-react-jsx-self";
...
     throwIfNamespace = true,
     useSpread,
     runtime = "classic",
     importSource,
   } = opts;
 
+  // TODO: (Babel 8) Remove setting these defaults
+  if (runtime === "classic") {
+    pragma = pragma ?? "React.createElement";
+    pragmaFrag = pragmaFrag ?? "React.Fragment";
```
Comment (nicolo-ribaudo) [inline packages/babel-preset-react/src/index.js]: Since we are trying to restore the old behavior, we should probably use `||` again.

Comment (existentialism) [reply packages/babel-preset-react/src/index.js]: :+1:

**Comment by nicolo-ribaudo** [comment]
(PR title edited otherwise it was cropped in the commit message)

**Comment by kentcdodds** [comment]
Are there plans to release this tonight or should I try and get things working with an older version?

Sorry, I hate to ask, I just don't know the release schedule/process here.

**Comment by nicolo-ribaudo** [comment]
Published in 7.9.4

**Comment by kentcdodds** [comment]
Thank you so much @nicolo-ribaudo! I'm sorry to bug you about it. Thank you all for the work you do!

**Comment by existentialism** [comment]
@kentcdodds sorry, fell asleep early last night :)

**Pull request merged**